test/services/katello/host_trace_manager_test.rb

```diff
@@ -0,0 +1,65 @@
+require 'katello_test_helper'
+
+unless Katello.with_remote_execution?
+  class JobInvocationComposer
+  end
+end
+
+module Katello
+  class HostTraceManagerTest < ActiveSupport::TestCase
+    def setup
+      @host1 = FactoryBot.create(:host)
```

Comment:
I think we can just use the fixtures `hosts(:one)` and two like we did previously

Reply:
Updated to use the fixtures instead of making more hosts.